translations/en.json

```diff
@@ -738,6 +738,7 @@
     "join-decentralised-communities": "Join Decentralized Communities",
     "http-gateway-error": "Oops, request failed!",
     "sign-request-failed": "Could not sign message",
+    "simple": "Simple",
```

Comment:
I think we can remove the translations as only used in the preview screen that we want to remove